WISE-Community/WISE
# Removed print lines.

I also found and fixed an issue with Run and Project getMaxID 
How to verify:
1. Login in with Teacher Account
2. Navigate to the Authoring Tool
3. Create a new Project - This process should succeed even if you had no previous projects[bug]
4. Log out of the Teacher Account
Check the debug console to make sure that the print statements are not outputted.

Closes #2016
